src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -219,20 +219,24 @@ private void rejectHouseholdIndividual(CaseContainerDTO caseDetails) {
     }
   }
 
+  private void rejectHouseholdIndividual(CaseDTO caseDetails) {
+    if (caseDetails.getCaseType().equals(CaseType.HI.name())) {
+      log.with(caseDetails.getId())
+          .info("Case is not suitable as it is a household individual case");
+      throw new ResponseStatusException(HttpStatus.FORBIDDEN, "Case is not suitable");
```

Comment:
should be BAD_REQUEST

Reply:
This method is exactly the same as an existing one but with a different method signature - it is an overloaded method. So the error thrown is existing code. I can change it if you like?

Reply:
Btw I have now removed the overloaded method and just changed the existing one to take a CaseDTO instead of a CaseContainerDTO.